popup.css

```diff
@@ -0,0 +1,922 @@
+  margin-right: 5px;
+}
+
+#detail figure img {
+  margin: 20px 0 15px 0;
+  width: 100%;
+  height: 181px;
+}
+
+#modal-close {
+  position: absolute;
+  right: 10%;
+  top: 4%;
+  color: #fff;
+  font-size: 20px;
+  z-index: 10;
+  border: none;
+  background: transparent;
+  background-color: rgba(255, 255, 255, 0.16);
+}
+
+.languages li {
+  color: #f5f6ff;
+  padding: 20px;
+  padding-top: 1vh;
+  font-size: 18px;
+}
+
+.frameworks li {
+  color: #f5f5ff;
+  padding-top: 0;
+  font-size: 18px;
+}
+
+.frameworks ul {
+  display: flex;
+  list-style-type: none;
+  flex-direction: column;
+  padding: 0;
+}
+
+.tags li {
+  list-style-type: none;
+  padding-right: 15px;
+  padding-left: 15px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  align-items: center;
+  font-weight: 500;
+  border-left: 2px solid #555;
+}
+
+.skills li {
+  padding-bottom: 20px;
+  padding-top: 0;
+}
+
+.social-media ul > li {
+  margin: 2vh;
+}
+
+.social-media-bottom ul > li {
+  margin: 2vh;
+}
+
+.menu_items li:nth-child(3) {
+  margin: 30px 0;
+}
+
+.menu_items li:nth-child(5) {
+  margin: 30px 0;
+}
+
+#detail .tags li:first-child {
+  border-left: 2px solid #555;
+}
+
+#detail .tags li:last-child {
+  border-right: 2px solid #555;
+}
+
+#button-group {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 2vh 0;
+}
+
+#button-group button span:nth-child(2) img {
+  width: 1em;
+  height: 1em;
+}
+
+.demo-button {
+  border: none;
+  width: 120px;
+  text-align: start;
+  background: #7d73fc;
+  height: 50px;
+  color: #edecff;
+  font-size: 17px;
+  padding: 8px;
+}
+
+.source-button {
+  border: none;
+  width: 140px;
+  margin: 2vh 0;
+  text-align: start;
+  background: #7d73fc;
+  height: 50px;
+  color: #edecff;
+  font-size: 17px;
+  padding: 7px;
+}
+
+.menu_items {
+  position: absolute;
+  left: 0;
+  background-color: #191765;
+  background-image: url('/images/shape2.png'), url('/images/shape1.png');
+  background-repeat: no-repeat, no-repeat;
+  background-position: left 4.5em, right bottom;
+  background-size: 13em, 11em;
+  height: 100vh;
+  width: 100%;
+  z-index: 1;
+  top: 0;
+  list-style-type: none;
+  display: none;
+  margin-left: 0;
+  padding-left: 0;
+}
+
+.menu_items li span {
+  color: #a1a3a7;
+}
+
+#toggleBtn {
+  background-color: transparent;
+  border: none;
+}
+
+#btn_close {
+  position: relative;
+  left: 92%;
+  background-color: transparent;
+  color: white;
+  font-size: 26px;
+  border: none;
+  margin-bottom: 50px;
+}
+
+.headlines h3 {
+  text-align: center;
+  margin-bottom: 5vh;
+}
+
+.languages h3 {
+  text-align: center;
+}
+
+.frame {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+@media screen and (max-width: 768px) {
+  .brand {
+    text-align: center;
+    padding: 6px;
+  }
+
+  .brand p {
+    margin-right: 0;
+  }
+
+  .work_first {
+    display: none;
+  }
+
+  .recent-desk {
+    display: none;
+  }
+
+  .projects > hr {
+    width: 300px;
+  }
+
```

Comment:
- [ ] Kindly, make sure that your design for the popup window for the mobile version looks exactly like the original design from Figma.
take a look at the table below:

|Figma Design|Your Design|
|---------------|-------------|
|![image](https://user-images.githubusercontent.com/71874425/189421941-6e6ad77e-098d-4d13-b8d1-824e1845844e.png)|![image](https://user-images.githubusercontent.com/71874425/189422247-fd384011-10ed-408e-aad9-8983e9f1520f.png)|